source/rust_verify_test/tests/lifetime.rs

```diff
@@ -149,8 +149,8 @@ test_verify_one_file! {
             x
         }
         proof fn test<A>(tracked x: A) {
-            let s = id(Option::Some(x)).is_None();
-            let s = id(Option::Some(x)).is_None();
+            let s = builtin::is_variant(id(Option::Some(x)), "None");
+            let s = builtin::is_variant(id(Option::Some(x)), "None");
```

Comment:
Why are these change necessary?

Reply:
It has to do with whether or not the spec function allows proof arguments ... normal spec functions don't, but apparently field access & variant checks do. The test needs something that does accept proof arguments.

`is_None` is a "normal" spec function now, so the test can't work while using it. On the other hand, `builtin::is_variant` lowers to the same VIR as is_None used to lower

The behavior is (and was) a bit inconsistent and could maybe use a revisit later.

Reply:
Maybe add this as a comment to that change? (Not critical, though.)